### ISSUE TITLE
[bug] data import tool

### ISSUE THREAD

**Issue by snvora** [frappe/erpnext#12757]
When downloading the template, earlier there used to be select all / unselect all button. which is now missing

![capture](https://user-images.githubusercontent.com/10496564/35774548-cf65d92e-0998-11e8-9ddd-6ffd6c2581ec.PNG)

**Comment by kennethsequeira** [comment]
If you simply want to select all the columns just select **All** from the **Select Column** dropdown option.

![screen shot 2018-02-05 at 11 19 47 am](https://user-images.githubusercontent.com/33246109/35790099-8b99dd84-0a66-11e8-8d4b-ed703c6c4f94.png)

**Comment by snvora** [comment]
I understand your suggestion.

But atleast unselect all button will be useful.

It's important where only few fields need to be updated.

**Comment by kennethsequeira** [comment]
Yup, that does make sense. A **Unselect All** button would be useful.
